src/AlignmentVault.sol

```diff
@@ -513,14 +540,17 @@ contract AlignmentVault is Ownable, Initializable, ERC721Holder, ERC1155Holder,
             _buildRemoveLiquidityParams(positionId, tokenIds, vTokenPremiumLimit, liquidity, amount0Min, amount1Min)
         );
         _WETH.withdraw(_WETH.balanceOf(address(this)));
+
+        // review: on full withdraw should we burn the position & remove from set? could be a try catch for burn()
```

Comment:
If the position isn't automatically burned or transferred upon full withdrawal, we should leave it in the set as it can be added to later.